packages/release/activate-configured.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Isolate configured.target
+After=preconfigured.target host-containerd.service
```

Comment:
`After` / `WantedBy` `preconfigured.target` is odd but seems right.

However the dependency on `host-containerd.service` does not seem correct; ideally we'd let that be started in phase two as part of `configured.target`.

Reply:
Done